### PR TITLE
Plugin: Create and sign plugin manifest

### LINKED ISSUE #141: Plugin: Signing of plugin binaries and files
**What would you like to be added**:
Signing of plugin manifest.

**Why is this needed**:
Additional security to make sure plugin binaries and files haven't been tampered with.

### PR DESCRIPTION
**Pull request by marefr**
Closes #141

### CODE REVIEW

**Comment by aknuds1** [comment]
Checking this out now.

**Comment by aknuds1** [comment]
Ah yeah, have to look at your build-pipeline PR first.

**Comment by aknuds1** [comment]
How about we make a candidate version of from your build pipeline PR and test it as part of this PR?

**Comment by aknuds1** [comment]
I pushed a change to use version 0.4.12 of grabpl, which I uploaded with symlink changes, seems to work! Shall we merge the symlink changes and publish 0.4.12?

File: .circleci/config.yml [hunk 1]
```diff
@@ -15,6 +15,18 @@ defaults: &defaults
...
     - image: circleci/node:12.13.1-stretch
 
+commands:
+  install-grabpl:
+    description: "Install Grafana build pipeline tool"
+    steps:
+      - run:
+          name: "Install Grafana build pipeline tool"
+          command: |
+            VERSION=0.4.12
```
Comment (aknuds1) [inline .circleci/config.yml:25]:
```suggestion
            VERSION=0.4.20
```

**Comment by marefr** [comment]
Confirmed Grafana loads and put signed badge on the plugin by adding the backend property to plugin.json:
image

**Pull request merged**